index.html

```diff
@@ -1153,13 +1149,13 @@ <h4>Proof Verification (eddsa-jcs-2022)</h4>
           </p>
 
           <ol class="algorithm">
-            <li>
-Let `publicKeyBytes` be the result of retrieving the
+          <li>
+Let |publicKeyBytes| be the result of retrieving the
 public key bytes associated with the
-`options`.`verificationMethod` value as described in the
-Data Integrity [[VC-DATA-INTEGRITY]] specification,
-<a data-cite="vc-data-integrity#algorithms">
-Section 4: Retrieving Cryptographic Material</a>.
+|options|.|verificationMethod| value as described in the
+[[[controller-document]]] specification,
+<a data-cite="controller-document#retrieve-verification-method">
+Section: Retrieve Verification Method</a>.
```

Comment:
Similar to the preceding... `cid`spec?
```suggestion
|options|.|verificationMethod| value as described in the
<a data-cite="controller-document#retrieve-verification-method">Retrieve
Verification Method</a> section of the [[[controller-document]]] specification.
```